code/__DEFINES/tools.dm

```diff
@@ -30,10 +30,10 @@
 #define WELDER_REPAIR_SUCCESS_MESSAGE		to_chat(user, "<span class='notice'>You repair the damage on [src]!</span>")
 #define WELDER_ATTEMPT_SLICING_MESSAGE		user.visible_message("<span class='notice'>[user] begins slicing through [src]...</span>", "<span class='notice'>You begin slicing through [src]...</span>", "<span class='warning'>You hear welding.</span>")
 #define WELDER_SLICING_SUCCESS_MESSAGE		to_chat(user, "<span class='notice'>You slice clean through [src]!</span>")
-#define WELDER_ATTEMPT_FLOOR_SLICE_MESSAGE	user.visible_message("<span class='notice'>[user] begins slicing [src] free from [get_turf(src)]...</span>", "<span class='notice'>You begin slicing [src] free from [get_turf(src)]...</span>", "<span class='warning'>You hear welding.</span>")
-#define WELDER_FLOOR_SLICE_SUCCESS_MESSAGE	to_chat(user, "<span class='notice'>You slice [src] clear of [get_turf(src)]!</span>")
-#define WELDER_ATTEMPT_FLOOR_WELD_MESSAGE	user.visible_message("<span class='notice'>[user] begins welding [src] to [get_turf(src)]...</span>", "<span class='notice'>You begin welding [src] to [get_turf(src)]...</span>", "<span class='warning'>You hear welding.</span>")
-#define WELDER_FLOOR_WELD_SUCCESS_MESSAGE	to_chat(user, "<span class='notice'>You weld [src] to [get_turf(src)]!</span>")
+#define WELDER_ATTEMPT_FLOOR_SLICE_MESSAGE	user.visible_message("<span class='notice'>[user] begins welding the [src] to [get_turf(src)]...</span>", "<span class='notice'>You begin welding the [src] to [get_turf(src)]...</span>", "<span class='warning'>You hear welding.</span>")
+#define WELDER_FLOOR_SLICE_SUCCESS_MESSAGE	to_chat(user, "<span class='notice'>You weld [src] to [get_turf(src)]!</span>")
+#define WELDER_ATTEMPT_FLOOR_WELD_MESSAGE	user.visible_message("<span class='notice'>[user] begins slicing the [src] free from [get_turf(src)]...</span>", "<span class='notice'>You begin slicing the [src] free from [get_turf(src)]...</span>", "<span class='warning'>You hear welding.</span>")
+#define WELDER_FLOOR_WELD_SUCCESS_MESSAGE	to_chat(user, "<span class='notice'>You slice [src] clear of [get_turf(src)]!</span>")
```

Comment:
I don't think we should swap the text here.

For example `WELDER_ATTEMPT_FLOOR_WELD_MESSAGE` shouldn't say  `You begin slicing the...`

I think the issue is in this code block in [field_generator.dm](code/modules/power/singularity/field_generator.dm):
```
/obj/machinery/field/generator/welder_act(mob/user, obj/item/I)
	. = TRUE
	if(state == FG_UNSECURED)
		to_chat(user, "<span class='warning'>[src] needs to be wrenched to the floor!</span>")
		return
	if(!I.tool_use_check(user, 0))
		return
	if(state == FG_SECURED)
		WELDER_ATTEMPT_FLOOR_SLICE_MESSAGE
	else if(state == FG_WELDED)
		WELDER_ATTEMPT_FLOOR_WELD_MESSAGE
	if(I.use_tool(src, user, 20, volume = I.tool_volume))
		if(state == FG_SECURED)
			WELDER_FLOOR_SLICE_SUCCESS_MESSAGE
			state = FG_WELDED
		else if(state == FG_WELDED)
			WELDER_FLOOR_WELD_SUCCESS_MESSAGE
			state = FG_SECURED
```
If we swap the messages here, they should be good.

Other files that reference the floor slice message are:
code/game/objects/structures/reflector.dm:		WELDER_ATTEMPT_FLOOR_SLICE_MESSAGE
code/game/machinery/mass_driver.dm:		WELDER_ATTEMPT_FLOOR_SLICE_MESSAGE
code/modules/recycling/sortingmachinery.dm:	WELDER_ATTEMPT_FLOOR_SLICE_MESSAGE
code/modules/recycling/disposal.dm:	WELDER_ATTEMPT_FLOOR_SLICE_MESSAGE
code/modules/power/singularity/emitter.dm:		WELDER_ATTEMPT_FLOOR_SLICE_MESSAGE
code/modules/power/singularity/field_generator.dm:		WELDER_ATTEMPT_FLOOR_SLICE_MESSAGE

Reply:
Oh wow yeah, didn't even see that